Examples/Python/python/acts/examples/itk.py

```diff
@@ -297,6 +297,8 @@ def itkSeedingAlgConfig(inputSpacePointsType: InputSpacePointsType):
     # variables that do not change for pixel and strip SPs:
     zMax = 3000 * u.mm
     zMin = -3000 * u.mm
+    zMinLastLayer =  2700 * u.mm
+    zMaxLastLayer =  -2700 * u.mm
```

Comment:
why do we have these values as +/- 2800 in the default configuration, but then we have them +/- 2700 for the ITk case? 

Can't they directly be +/- 2700 in the configs as well?